requirements.txt

```diff
@@ -1,5 +1,9 @@
-pandas~=2.2.0
-requests~=2.31.0
 beautifulsoup4~=4.12.3
-selenium~=4.16.0
-lxml~=5.1.0
\ No newline at end of file
+selenium~=4.17.2
+fake-useragent~=1.4.0
```

Comment:
As before, we don't need to fake the useragent, should follow the SEC guidance https://www.sec.gov/os/webmaster-faq#code-support